src/pseudopeople/configuration/generator.py

```diff
@@ -11,6 +11,7 @@
 
 # Define non-baseline default items
 # NOTE: default values are defined in entity_types.RowNoiseType and entity_types.ColumnNoiseType
+SSA_SSN_COLUMN = [column for column in FORMS.ssa.columns if column.name == "ssn"][0]
```

Comment:
Instead of this,

```
from pseudopeople.schema_entities import COLUMNS

SSA_SSN_COLUMN = COLUMNS.ssn
```

Reply:
Ah, for some reason I thought there could be different noise functions applied to the same column in different datasets. I'll fix this.